Tools/Scripts/libraries/webkitscmpy/webkitscmpy/test/svn_unittest.py

```diff
@@ -323,7 +323,7 @@ def test_info(self):
         with mocks.remote.Svn():
             self.assertDictEqual({
                 'Last Changed Author': 'jbedard@apple.com',
-                'Last Changed Date': datetime.utcfromtimestamp(1601665100 - timedelta(hours=7).seconds).strftime('%Y-%m-%d %H:%M:%S'),
+                'Last Changed Date': datetime.fromtimestamp(1601665100 - timedelta(hours=7).seconds, timezone.utc).strftime('%Y-%m-%d %H:%M:%S'),
```

Comment:
This feels weird, taking a hard-coded timestamp and subtracting a delta. But it's not a change from the status quo.